src/bindings/go-elektra/go.mod

```diff
@@ -1,3 +1,5 @@
 module go.libelektra.org
 
 go 1.20
+
+replace github.com/ElektraInitiative/libelektra/src/bindings/go-elektra v0.10.0 => go.libelektra.org v0.10.0
```

Comment:
I don't really have any experience with Go, but this seems backwards to me.

AFAIK `replace` is intended for overriding libraries with local copies, like this

https://github.com/ElektraInitiative/libelektra/blob/567bb7fe5a89de0812da3d9452f61bcf70369fa3/src/tools/elektrad/go.mod#L11

Also since _this is_ the `go.libelektra.org` module (as defined in line 1), I don't believe a `replace` makes sense here. Using `go.libelektra.org/foo` should already be interpreted as just use `foo` from the local files.

So I don't think this PR is necessary at all.

To clarify what I said in https://github.com/ElektraInitiative/libelektra/pull/4927#discussion_r1194377439:

I assumed, we may have used something like

```go
import (
   elektra "https://github.com/ElektraInitiative/libelektra/tree/master/src/bindings/go-elektra/kdb"
)
```

in some go files (e.g. some example intended to run outside the repo), since `go.libelektra.org` still pointed at the wrong URL. If we had that, we should have changed it back to 

```go
import (
   elektra "go.libelektra.org/kdb"
)
```

_after_ the release.


However, since we only have the `go.libelektra.org` module itself (`src/bindings/go-elektra`) and `elektrad`, which uses `replace` with a relative path, that's not the case. We use `go.libelektra.org`, but building our go code never actually tries to resolve `go.libelektra.org`. It's either the current module (which doesn't need resolving) or is explicitly replaced with a relative path.
Therefore, AFAIK we can just release 0.10.0 as-is, without any extra changes before or after the release and both our code and external code should work (again).

Reply:
Also since `github.com/ElektraInitiative/libelektra/src/bindings/go-elektra` isn't used anywhere in this module, the line is useless anyway.